### ISSUE TITLE
Is there any plans to move to gymnasium?

### ISSUE THREAD

**Issue by Mayankm96** [Denys88/rl_games#261]
Hi Developers!

It seems RL-Games is still using `gym` instead of the more modern `gymnasium`. This seems to be hardcoded in some places of the code where the type is checked against gym action spaces.

Example:
> https://github.com/Denys88/rl_games/blob/master/rl_games/common/experience.py#L308-L319

Are there any plans to move to gymnasium, or to make the type-checks lenient to check if it is `gym` or `gymnasium` space?

And if not, then is there a way to specify spaces without using either of them?

Thanks!

**Comment by Denys88** [comment]
Yes, I even have branch where i am testing it. But there a lot of interesting projects which still use gym so I want to make sure  they will continue working.
I think I can finish it in a 3 three weeks.

**Comment by Denys88** [comment]
hi @Mayankm96  I am still working on it because there a lot of strange envs.
